src/EnergyPlus/OutputProcessor.hh

```diff
@@ -561,8 +606,8 @@ namespace OutputProcessor {
     void InitializeOutput(EnergyPlusData &state);
 
     void SetupTimePointers(EnergyPlusData &state,
-                           std::string const &IndexKey, // Which timestep is being set up, 'Zone'=1, 'HVAC'=2
-                           Real64 &TimeStep             // The timestep variable.  Used to get the address
+                           eTimeStepType const &IndexKey, // Which timestep is being set up, 'Zone'=1, 'HVAC'=2
```

Comment:
SetupTimePointers now also must take this enum type.

Reply:
Why is this variable passed by reference?

Reply:
Just because it was when it was a string.  Will change it to be passed by value.